tox.ini

```diff
@@ -0,0 +1,15 @@
+# tox (https://tox.readthedocs.io/) is a tool for running tests
+# in multiple virtualenvs. This configuration file will run the
+# test suite on all supported python versions. To use it, "pip install tox"
+# and then run "tox" from this directory.
+
+[tox]
+envlist = py{35}-django{22}, py{38}-django{22,30}
+
+[testenv]
+deps =
+	-r test_requirements.txt
```

Comment:
Oh I also had to remove the space after `-r` otherwise I was getting an error ` No such file or directory: ' test_requirements.txt'`.